perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -206,8 +213,28 @@ public static String join(Iterable<?> collection, String separator) {
 							throw new ConsistencyErrorException("Creating existing extSource: " + additionalExtSourceName);
 						}
 					}
+					UserExtSource userExtSource = new UserExtSource(additionalExtSource, additionalExtLoa, additionalExtLogin);
+
+					List<Attribute> attributes = new ArrayList<>();
+					// Set ues attribute
+					if (additionalExtLoginAndAttribute.length > 1) {
+						List<String> uesAttributesWithValues =  new LinkedList<>();
+						uesAttributesWithValues.addAll(Arrays.asList(additionalExtLoginAndAttribute));
+						uesAttributesWithValues.remove(0);
+						for (String uesAttributeWithValue : uesAttributesWithValues) {
+							String[] uesAttribute = uesAttributeWithValue.split("=");
+							try {
+								Attribute attribute = new Attribute(sess.getPerun().getAttributesManager().getAttributeDefinition(sess, uesAttribute[0]), uesAttribute[1]);
```

Comment:
I think this will work only for single value attributes. @stavamichal do we want to support also multivalue attributes?

Reply:
@balcirakpeter Do we already have ues attribute with multi-value as logins? If yes, then we need to fix it too. Which means to change format for the multivalue attribute. For example we can use something like "|login,urn=value1;value2,urn2=value1|1" (at least if these characters are not in the login itself and they are not used for parsing the whole string yet).

Reply:
If you will be changing this one, please after the change send me re-request for review. Thank you.

Reply:
Yes, we have such ues attribute on eduTEAMS instances called additionalIdentifiers.

Reply:
Ok, so I've updated extraction to support multivalue attributes and re-request your reviews.

Reply:
In such a case please @metodej prepare this code to be able to process multi-value attributes. Don't need to use my suggestion.